toolchain/mfc/test/test.py

```diff
@@ -180,7 +181,7 @@ def _handle_case(case: TestCase, devices: typing.Set[int]):
                 raise MFCException(f"Test {case}: {msg}")
 
     if ARG("test_all"):
-        case.delete_output()
+        # case.delete_output()
```

Comment:
might want to put this back?

Reply:
It breaks the 1D bubble cases. Not sure why maybe its deleting the D folder I'm creating.

Reply:
perhaps associated with current bug in CI that if it has to 'retry' a simulation, it will try to create a directory that already exists